subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/suite/filter/TestsFilter.kt

```diff
@@ -42,7 +47,8 @@ interface TestsFilter {
         val name: String,
         val annotations: List<AnnotationData>,
         val deviceName: DeviceName,
-        val api: Int
+        val api: Int,
+        val flakiness: Flakiness
```

Comment:
When we create that model we already parsed annotations. So, I decided to reuse the result of parsing